### PR TITLE
luaradio: depend on luajit-openresty

### PR DESCRIPTION
**Pull request by carlocab**
Follow up to #68787.

-----

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

**Comment by carlocab** [comment]
@fxcoudert interested in your thoughts on this, since I've taken a somewhat ugly approach to the dependencies. The issue is that `luajit-openresty` is `keg-only`, but `luaradio` looks for `luajit` in the `PATH`. 

I haven't figured out how to tell it where to look for `luajit` when it's not on `PATH`. Changing the make variables with explicit path references didn't work...

**Comment by SMillerDev** [comment]
Couldn't we do a `env_script`?

File: Formula/luaradio.rb [hunk 1]
```diff
@@ -27,7 +28,21 @@ def install
...
-      system "make", "install", "PREFIX=#{prefix}"
+      system "make", "install", "PREFIX=#{libexec}"
+
+      env = {
+        PATH:      "#{Formula["luajit-openresty"].opt_bin}:$PATH",
+        LUA_PATH:  "#{libexec}/lib/lua/5.1/?.lua${LUA_PATH:+:$LUA_PATH}",
+        LUA_CPATH: "#{libexec}/lib/lua/5.1/?.so${LUA_CPATH:+:$LUA_CPATH}",
+      }
+
+      Dir["#{libexec}/bin/*"].each do |path|
```
Comment (SMillerDev) [inline Formula/luaradio.rb:39]: Isn't this the same as `env_script_all_files` or whatever that one is called?

Comment (carlocab) [reply Formula/luaradio.rb:39]: Looks like it; I just based this off another formula. Let me fix that.

Comment (carlocab) [reply Formula/luaradio.rb:39]: Actually, it's not quite the same. I'll need to change other parts of the install block to use this. Might break the binaries if relative paths are baked in. Still, worth a try.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**